test/updeep-spec.js

```diff
@@ -142,4 +142,13 @@ describe('updeep', () => {
     const result = u({ created: date }, {})
     expect(result).to.eql({ created: date })
   })
+
+    it('omit properties via u.omitted', () => {
+        const result = u({
+            a: u.omitted,
+            b: i => i + 1,
+        }, { a: 1, b: 2 })
+
+        expect(result).to.eql({ b: 3 });
+    })
```

Comment:
Please include a test for a nested object and for an array (what should it do if you omit in an array?)

Reply:
tests added.

The `_.omitted` should skip the property value, no matter if it's a scalar or an array or object, so we should be good there.

The undefined case is if one does:

```
u({ 3: u.omitted })([ 1,2,3,4,5 ]);
```

I could add logic to take care of that case, basically do

```
  if (Array.isArray(defaultedObject)) {
    return updateArray(resolvedUpdates, defaultedObject).filter( item => item !== innerOmitted )
  }
```

 but it just doesn't feel like something anybody would do.

Reply:
Yeah, I guess the question is, what should the lib do... throw? Create a sparse array? filter it out as you demonstrate there? 

Maybe throwing for now in `updateArray` would work.